ui/tests/integration/components/pgp-file-test.js

```diff
@@ -81,12 +81,15 @@ module('Integration | Component | pgp file', function(hooks) {
 
     await render(hbs`{{pgp-file index=index key=key onChange=(action change)}}`);
     await triggerEvent('[data-test-pgp-file-input]', ...event);
+    await settled();
     await click('[data-test-text-toggle]');
     assert.equal(findAll('[data-test-pgp-file-textarea]').length, 1, 'renders the textarea on toggle');
+
     assert.equal(
       find('[data-test-pgp-file-textarea]').textContent.trim(),
       this.lastOnChangeCall[1].value,
       'textarea shows the value of the base64d key'
     );
+    await settled();
```

Comment:
curious about this too -- why wait for things to settle after making an assertion?

Reply:
Whoops, I think this was left in from when we were looking at fixing the test.